finance-service/src/main/kotlin/com/ajaxproject/financeservice/model/MongoFinance.kt

```diff
@@ -0,0 +1,47 @@
+package com.ajaxproject.financeservice.model
+
+import com.ajaxproject.financeservice.model.MongoFinance.Companion.COLLECTION_NAME
+import com.ajaxproject.internalapi.finance.commonmodels.FinanceMessage
+import com.ajaxproject.internalapi.finance.commonmodels.FinanceType
+import org.bson.types.ObjectId
+import org.springframework.data.annotation.Id
+import org.springframework.data.annotation.TypeAlias
+import org.springframework.data.mongodb.core.mapping.Document
+import java.util.Date
+
+@TypeAlias("Finances")
+@Document(value = COLLECTION_NAME)
+data class MongoFinance(
+
+    @Id
+    val id: ObjectId = ObjectId(),
```

Comment:
Why not to make id nullable? Mongo will generate ObjectId automatically, when saving with id = null